src/main/java/com/chat/yourway/model/Topic.java

```diff
@@ -48,19 +48,17 @@ public class Topic {
   @Column(name = "created_at", nullable = false)
   private LocalDateTime createdAt;
 
-  @ManyToMany
+  @ManyToMany(fetch = FetchType.EAGER)
```

Comment:
Why do you use FetchType.EAGER in here?

Reply:
Because I have a problem with lazy initialization in a one test case, where mapper try to working with a size of list tags. It's embedded implementation, so first time I decided to set eager type, but when there is not more tasks, we can fix problem by implementing own mapper for list of tags in topic. So it's temporary solution.